kahuna/public/js/components/gu-lazy-table/gu-lazy-table.js

```diff
@@ -92,24 +94,59 @@ lazyTable.controller('GuLazyTableCtrl', [function() {
             filter(({$start, $end}) => $start <= $end).
             distinctUntilChanged(({$start, $end}) => `${$start}-${$end}`);
 
+        // Placeholders
+        const placeholderExtraCount$ = mult$(columns$, preloadedRows$);
+        const placeholderRangeStart$ = max$(sub$(loadedRangeStart$, placeholderExtraCount$), 0);
+        const placeholderRangeEnd$ = min$(add$(loadedRangeEnd$, placeholderExtraCount$),
+                                          itemsCount$);
+
+        const placeholderIndexes$ = combine$(
+            items$, placeholderRangeStart$, placeholderRangeEnd$,
+            (items, placeholderRangeStart, placeholderRangeEnd) => {
+                let indexes = [];
+                for (let i = placeholderRangeStart; i < placeholderRangeEnd; i++) {
```

Comment:
I think @akash1810 may have written a range elsewhere using generators actually (?).

Happy to replace if so and no performance hit.